packages/react-instantsearch-hooks/src/lib/useInstantSearch.ts

```diff
@@ -7,7 +7,6 @@ import version from '../version';
 
 import { useForceUpdate } from './useForceUpdate';
 import { useStableValue } from './useStableValue';
-import { warn } from './warn';
```

Comment:
this was the only use of warn, it can be removed, unless we plan on having it again for another reason

Reply:
I think we can keep it since it gets out of the final bundle anyway. It'll likely be useful for other purposes.